ui/src/components/navbar.tsx

```diff
@@ -94,6 +94,7 @@ export class Navbar extends Component<any, NavbarState> {
         <button
           class="navbar-toggler"
           type="button"
+          aria-label="menu"
```

Comment:
I'm pretty noobish on aria stuff, but I do wanna make the entire app screenreader capable at some point. If there's other places this can be done, that'd be a huge help.

Reply:
Wherever i see the need i will add aria helpers and other attributes to improve accessibility (using `<main>` and `<aside>` also help with this already, as you give semantic meaning to elements.